diagnostic.md

```diff
@@ -32,7 +32,23 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
+```HTML
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML/CSS diagnostic</title>
+  </head>
+  <body>
+    <ul>
+      <li> Item one
+      <li> Item two
+      <li> Item three
+    </ul>
 
+  </body>
+</html>
+```
```

Comment:
Best practice is to include closing `</li>`. They will throw weird errors when you least expect.